src/components/OperationsList/AccountCell.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { getCryptoCurrencyIcon } from '@ledgerhq/live-common/lib/react'
+import type { Currency } from '@ledgerhq/live-common/lib/types'
+import Box from 'components/base/Box'
+
+const Cell = styled(Box).attrs({
+  px: 4,
+  horizontal: true,
+  alignItems: 'center',
+})`
+  width: 150px;
+  overflow: hidden;
+`
+
+type Props = {
+  currency: Currency,
+  accountName: string,
+}
+
+class AccountCell extends PureComponent<Props> {
+  render() {
+    const { currency, accountName } = this.props
+    const Icon = getCryptoCurrencyIcon(currency)
+    return (
+      <Cell horizontal flow={2}>
+        <Box alignItems="center" justifyContent="center" style={{ color: currency.color }}>
+          {Icon && <Icon size={16} />}
```

Comment:
condition on `Icon` can be put directly on containing Box

Reply:
also, in future, we probably want to always have an <Icon> and no icon would take same space. also probably make that Icon is directly a component exported by live-common and you give me the currency as prop

Reply:
note that i've not really touched this part, so probably leave it for now & i think we'll refactor to this bigger idea?